src/utils/sortPeople.tsx

```diff
@@ -0,0 +1,45 @@
+import { SortParam } from '../types/SortParam';
+import { Person } from '../types';
+
+export const sortPeople = (
+  people: Person[], sortBy: string, order: string,
+) => {
```

Comment:
It's better to use a more descriptive variable name than 'order' for clarity. Consider renaming it to 'sortOrder'.